package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gray-matter",
   "description": "Parse front-matter from a string or file. Fast, reliable and easy to use. Parses YAML front matter by default, but also has support for YAML, JSON, TOML or Coffee Front-Matter, with options to set custom delimiters. Used by metalsmith, assemble, verb and many other projects.",
-  "version": "4.0.3",
+  "version": "4.0.4",
```

Comment:
Please revert this. version changes are typically done by maintainers. Also, I personally like to play it safe with major version bumps, especially since incompatible node versions can easily sneak in.

Reply:
kk reverted that change :+1: